src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/RandomNumberGenerator.cs

```diff
@@ -288,17 +289,60 @@ public static string GetHexString(int stringLength, bool lowercase = false)
         /// <typeparam name="T">The type of span.</typeparam>
         public static void Shuffle<T>(Span<T> values)
```

Comment:
Changes for cryptographic RNG may subject to more careful review for security requirements.